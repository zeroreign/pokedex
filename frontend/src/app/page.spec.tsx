import { render, screen } from '@testing-library/react'
import Home from './page'


describe('App', () => {
  it('renders a heading', () => {
    render(<Home />)
     
    const heading = screen.getByRole('heading', { level: 1 })
 
    expect(heading).toBeInTheDocument();
    expect(heading).toHaveTextContent('Pokedex');
  })

  it('renders homepage unchanged', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  })
})