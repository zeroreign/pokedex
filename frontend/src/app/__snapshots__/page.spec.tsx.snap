// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders homepage unchanged 1`] = `
<div>
  <div>
    <h1>
      Pokedex
    </h1>
    <p>
      Welcome to the Pokedex App
    </p>
  </div>
</div>
`;
